core/src/main/scala/com/softwaremill/react/kafka/TopicPartitionPair.scala

```diff
@@ -0,0 +1,13 @@
+package com.softwaremill.react.kafka
+
+import org.apache.kafka.common.TopicPartition
+
+/**
+ * Exists to keep {@link TopicPartition} from Kafka out of our interfaces.
+ * @param topic
+ * @param partition
+ */
+case class TopicPartitionPair(topic: String, partition: Int) {
```

Comment:
I understand that you would like to hide direct Kafka API as much as possible, but I think that we shouldn't worry that much about stuff from the kafka-clients.jar. We already use `ConsumerRecord`, for example. This API is meant to be stable.